mutt/path.c

```diff
@@ -347,15 +347,22 @@ bool mutt_path_to_absolute(char *path, const char *reference)
   buf_printf(abs_path, "%s/%s", dirpath, path);
   FREE(&dirpath);
 
-  path = realpath(buf_string(abs_path), path);
-  if (!path && (errno != ENOENT))
+  char rpath[PATH_MAX + 1] = { 0 };
+  const char *result = realpath(buf_string(abs_path), rpath);
+  buf_pool_release(&abs_path);
+  if (result == NULL)
   {
-    mutt_perror(_("Error: converting path to absolute"));
-    buf_pool_release(&abs_path);
+    if (errno != ENOENT)
```

Comment:
I wonder if we should exclude this case at all. Is it common that `mutt_path_to_absolute` gets called with a non-existing path?

Reply:
This was introduced in #1293. I have no idea why. @flatcap ?